ledger/streaming/api/src/components.rs

```diff
@@ -2,10 +2,11 @@
 
 use mc_attest_core::VerificationReport;
 use mc_consensus_scp::QuorumSet;
+use mc_crypto_digestible::Digestible;
 use mc_transaction_core::BlockData;
 
 /// Wrapper for the components needed for block streaming.
-#[derive(Clone, Debug, PartialEq, Eq)]
+#[derive(Clone, Debug, Digestible, PartialEq, Eq)]
```

Comment:
This doesn't really need to be digestible IMO. Digesting is more for when dealing with things that are an immutable record that need to be verified later - these component objects are highly ephemeral and unlikely to be hashed/stored anywhere. With the blockstream, the hash will be changing with each new block/avr and the blocks/avrs already have digests consumers can compare downstream. Mishbehaving streaming nodes lying about quorum set are more easily detected by scanning those directly than inspecting a hash.

Reply:
@remoun and I discussed some changes to how we're storing data in S3 (and potentially the ledger) which should help this situation. There's a couple MCIPs that I need to write which will generate future PRs to update this code.